save-common/src/commonMain/kotlin/org/cqfn/save/core/config/TestConfig.kt

```diff
@@ -105,6 +105,16 @@ data class TestConfig(
         }
         return this
     }
+
+    /**
+     * Method, which validates all plugin configs and set default values, if possible
+     */
+    fun validateAndSetDefaults() {
+        logDebug("Start plugin validation for $location")
+        for (config in pluginConfigs) {
```

Comment:
`forEachIndexed` instead of calling `indexOf` on every iteration?

Reply:
Yes, it will be more appropriate, fixed